src/langcheck/augment/__init__.py

```diff
@@ -0,0 +1,5 @@
+from langcheck.augment import en
+
+__all__ = [
+    "en",
```

Comment:
From `langcheck.metrics` you can also directly call English metrics without `en` module. Could you also do that in `augment` for consistency? (I do not think it is a very good design, but let's re-design them altogether later)

Reply:
Import path updated